src/index.html

```diff
@@ -5,10 +5,92 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+
+    <!-- Google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" 
+      href="https://fonts.gstatic.com"
+      crossorigin="use-credentials"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
+    <!-- Connecting style.css -->
     <link rel="stylesheet" href="./style.css">
+
+    <title>Moyo header</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <nav class="navbar">
+        <ul class="navbar-nav">
+          <li class="nav-item">
+            <a href="#home"
```

Comment:
if u have 2 or less attributes leave them on one line

Reply:
fix everywhere, if u r postpone attributes don't leave one of them with opening tag